### ISSUE TITLE
Website Call To Action --> Conversion to Paying + Happy Customer

### ISSUE THREAD

**Issue by nelsonic** [mosteirodelandim/landim-website#68]
As noted in: https://github.com/mosteirodelandim/landim-website/issues/34#issuecomment-1736416537 the newly published site while slick, fast and attractive 🎉 
is devoid of a [**`Call to Action`**](https://www.optimizely.com/optimization-glossary/call-to-action) which means the site will not "convert" a casual browsing person to a "Lead" i.e. capturing some sort of personal detail e.g. `email` or `phone_number` so that a member of the team can follow-up and discuss potential requirements. 

What is _needed_ on a Wedding Website - if you want more sales and eventually higher margins and profitability ... is: 

# Todo

+ [ ] An ultra-low friction contact-us or register interest form that allows people to input their
  + [ ] `name`
  + [ ] `email_address`
  + [ ] `phone_number`
  + [ ] `message`
  > Personal data should be securely stored in your `CRM` where appropriate action can be taken. i.e. not stored in the "inbox" or _much_ worse @iteles (or another family member's) head. 
+ [ ] Availability Calendar #67
+ [ ] A call to "follow us on `X`" which will give you _some_ personal data point you can use to follow-up with the person.
  (which ever [social](https://github.com/dwyl/home/issues/29) [networking](https://github.com/dwyl/learn-react/issues/23) time-sucking society-destroying crap you're using to promote the venue ...)

**Comment by iteles** [comment]
You're absolutely right. The new site is MILES better than the old one and I'm SO HAPPY we deployed it @LuchoTurtle 🥳 

We need a call to action and yet, the owners can't decide on what that should be for now.
Whilst it's obvious to me that it should be events, there is some debate (and tension) around this.

Thanks for opening this issue.

Personally, I pour enough time into Landim and won't be doing the thinking around this but totally up for revising anything that gets done (the easier part) 🤷
